src/ui/idam-maintenance-ui/src/FamilyHubs.Idams.Maintenance.Core/ApiClient/ServiceDirectoryClient.cs

```diff
@@ -24,6 +24,12 @@ public ServiceDirectoryClient(HttpClient client, ICacheService cacheService, ILo
 #endif
     }
 
+    public ServiceDirectoryClient(HttpClient client, ILogger<ServiceDirectoryClient> logger)
```

Comment:
good spot.. I did this when I was looking at solutions. I'll revert it because I'm not using it.
Your second point is a good one and worth looking into as another tech debt